explainer.md

```diff
@@ -69,7 +69,7 @@ self.addEventListener("fetch", fetchEvent => {
       }
     }
     return fetch(fetchEvent.request);
-  }());
```

Comment:
I think these parens are needed here in order to do an immediately invoked async function.  We need to pass a promise to respondWith() which calling the async function will do.

Reply:
Thank you. I see the promise! Actually, I tried to execute these example codes in the explainer, but it doesn't work because of the js syntax error. 
We should fix this like `(async (_) => {...})()`, right? I will fix it later.